packages/roosterjs-editor-api/lib/format/setImageAltText.ts

```diff
@@ -2,7 +2,15 @@ import execFormatWithUndo from './execFormatWithUndo';
 import queryNodesWithSelection from '../cursor/queryNodesWithSelection';
 import { Editor } from 'roosterjs-editor-core';
 
-export default function setImageAltText(editor: Editor, altText: string): void {
+/**
+ * Set image alt text for all selected images at selection. If no images is contained
+ * in selection, do nothing.
+ * Alt text (alternative text) is a word or phrase that can be inserted as an
+ * attribute in an HTML document to tell viewers the contents of an image.
```

Comment:
Explain alt text according to https://www.w3schools.com/tags/att_img_alt.asp
And you can paste this url in comment for reference as well.